src/StackHCI/custom/stackhci.ps1

```diff
@@ -200,6 +200,8 @@ $ClusterScheduledTaskReadyState = "Ready"
 
 $ArcSettingsDisableInProgressState = "DisableInProgress"
 
+$AzAccountsModuleVersion="2.10.2"
```

Comment:
1. What will happen if module of different version has been installed on the machine?
2. How are we planning to keep these versions updated